data.sql

```diff
@@ -1,6 +1,6 @@
 /* Populate database with sample data. */
```

Comment:
![image](https://github.com/abenezer-addisu/Vet-clinic-database/assets/93039359/68b40315-a1b9-4b6c-b628-0d14bfede141)

- [ ] It would be nice if you remove all the placeholder texts from your readme file to make your readme look clean and professional 👍